index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>To Do-list App</title>
+</head>
+<body>
+  <div class="todo-container">
+    <div class=" list appContainer">
+      <div class=" todo-header">
+        <h2>
+           Today's todo
+        </h2>
+        <i class="fas fa-sync-alt"></i>
+      </div>
+     <div class="todo-input">
+       <input class="input-task" type="text" id="input" placeholder="to do item">
```

Comment:
- [ ] Kindy change your placeholder title to be 'Add to your list`, which is more descriptive of what the user should do, Also there is a refresh button missing in your design, below is a comparison with the design given;

Figma Design          |  Your design     
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/61408860/174428140-fc7acb30-b75d-47cd-9e9d-9a269e459c3e.png) |  <img width="748" alt="image" src="https://user-images.githubusercontent.com/61408860/174428107-e66b25ff-f8f3-4659-8dc5-4b7d5fc80620.png">